.github/workflows/ci.yml

```
@@ -86,8 +86,9 @@ jobs:
               uses: shivammathur/setup-php@v2
               with:
                   php-version: ${{ matrix.php-version }}
+                  tools: castor
```

As you support very old PHP versions, you can also use the official action to install a static binary of Castor embedding a modern PHP version

```suggestion

           - name: Setup castor
             uses: castor-php/setup-castor@v0.1.0
```